shell_integration/nautilus/syncstate.py

```diff
@@ -205,6 +205,8 @@ def check_registered_paths(self, filename):
     def get_file_items(self, window, files):
         # Show the menu extension to share a file or folder
 
+        if len(files) == 0:
+            return [] # Appears to happen occasionally
```

Comment:
I hate to nitpick here but I would (like @misch7 mentioned) prefer a blank line after the `return` (please also make sure there is no whitespace on that line).

Moreover, I would prefer to have the comment after the `if` line, not the `return` line.